Assets/HoloToolkit/Sharing/Prefabs/ManualIpConfigUtility.prefab

```diff
@@ -0,0 +1,3126 @@
+%YAML 1.1
```

Comment:
This is mainly for the HTK sharing, so that's why it is highly coupled.

I think uNET has a lot more flexibility and doesn't have a IP requirement seeing as they have their own matchmaking services built in.